connection.js

```diff
@@ -616,16 +647,19 @@ class Connection {
                  }
                 // now just before TimecontrolString is being tested, we again make sure it has the latest value
                 universalPeriodtimeTimecontrolString = String(t.time_control);/*
-                "fischer", "byoyomi", "canadian" */
-
-                // sanity check : if not fischer, not byoyomi, not canadian
-                if ((universalPeriodtimeTimecontrolString !== "fischer") && (universalPeriodtimeTimecontrolString !== "byoyomi") && (universalPeriodtimeTimecontrolString !== "canadian")) {
-                    conn_log ("error, could not find time control in " + t.time_control);
-                    return { reject : true, msg: "error, could not find time control in " + t.timecontrol};
+                "fischer", "byoyomi", "canadian", "simple" */
+
+                // sanity check : if not fischer, not byoyomi, not canadian, not simple
+                // time control peculiarities : 
+                // - fischer : doesnt have a minperiods or maxperiods
+                // - simple : doesnt have a maintime, only periodtime
+                if ((universalPeriodtimeTimecontrolString !== "fischer") && (universalPeriodtimeTimecontrolString !== "byoyomi") && (universalPeriodtimeTimecontrolString !== "canadian") && (universalPeriodtimeTimecontrolString !== "simple")) { // simple is part of allowed timecontrols
```

Comment:
the comments are fixed later on in UHMAEAT 2.2 in #129 

i think you'd rather want to refer to version 2.2 for review

again, if no critical bug here, i'd rather merge it as is so that we can merge next ones

Reply:
What's the point of partial commits if they're not cleaned up? Just rebase later.